src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -2515,6 +2515,23 @@ public override string ToString()
     }
 }
 
+public class NonSerializablePerson2
```

Comment:
Nit: I would prefer descriptive word instead of number '2', for example NonSerializablePersonForStress. Same for FamilyForStress, NonSerializablePersonSurrogateForStress